src/components/grid-aware/PartnersAndSponsorsBlock/PartnersAndSponsorsBlock.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./PartnersAndSponsorsBlock.module.css";
+
+const LogoPropType = PropTypes.shape({
+  key: PropTypes.number.isRequired,
```

Comment:
`key` is actually a special React prop [that is only meant to be used on array items](https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys). It's not a prop that is used by the item itself, but rather a prop used by the implicit array parent in order to track which child elements to re-render if the array contents change.

To quote the documentation that I linked to above:

> For example, if you extract a `ListItem` component, you should keep the key on the `<ListItem />` elements in the array rather than on the `<li>` element in the `ListItem` itself.

Also, the key isn't something that you necessarily need the person instantiating the top-level component to specify. It just needs to be some unique identifier amongst the array items. In think case, I think either the URL or the alt text should be unique, so you can use either.